keras/objectives.py

```diff
@@ -1,25 +1,22 @@
 from __future__ import absolute_import
-import theano
-import theano.tensor as T
-import numpy as np
 
+import numpy as np
+import theano.tensor as T
 
-if theano.config.floatX == 'float64':
-    epsilon = 1.0e-9
-else:
-    epsilon = 1.0e-7
+from keras import backend as K
+from keras.backend.common import _EPSILON as epsilon
 
 
 def mean_squared_error(y_true, y_pred):
-    return T.sqr(y_pred - y_true).mean(axis=-1)
+    return K.mean(K.square(y_pred - y_true))
```

Comment:
This is fixed. Please do not submit PRs that are incorrect or unfinished, post a Github issue instead.